src/Simulation/Simulators.Tests/ResourcesEstimatorTests.cs

```diff
@@ -101,5 +100,43 @@ public void ToTSVTest()
             Assert.Equal(2, cliffords.Length);
             Assert.Equal("2", cliffords[1]);
         }
+
+        /// <summary>
+        /// Verifies that for multiple separately traced operations, the final
+        /// statistics are cumulative.
+        /// </summary>
+        [Fact]
+        public void VerifyTracingMultipleOperationsTest()
+        {
+            ResourcesEstimator sim = new ResourcesEstimator();
+
+            Operation_1_of_2.Run(sim).Wait();
+            DataTable data1 = sim.Data;
+
+            Assert.Equal(1.0, data1.Rows.Find("CNOT")["Sum"]);
+            Assert.Equal(1.0, data1.Rows.Find("QubitClifford")["Sum"]);
+            Assert.Equal(1.0, data1.Rows.Find("T")["Sum"]);
+            Assert.Equal(0.0, data1.Rows.Find("R")["Sum"]);
+            Assert.Equal(0.0, data1.Rows.Find("Measure")["Sum"]);
+            Assert.Equal(2.0, data1.Rows.Find("Width")["Sum"]);
+
+            Operation_2_of_2.Run(sim).Wait();
+            DataTable data2 = sim.Data;
+
+            // Aggregated stats for both operations.
+            Assert.Equal(1.0 + 2.0, data2.Rows.Find("CNOT")["Sum"]);
+            Assert.Equal(1.0 + 1.0, data2.Rows.Find("QubitClifford")["Sum"]);
+            Assert.Equal(1.0 + 0.0, data2.Rows.Find("T")["Sum"]);
+            Assert.Equal(0.0 + 1.0, data2.Rows.Find("R")["Sum"]);
+            Assert.Equal(0.0 + 1.0, data2.Rows.Find("Measure")["Sum"]);
+            Assert.Equal(2.0 + 3.0, data2.Rows.Find("Width")["Sum"]);
+            Assert.Equal(System.Math.Max(2.0, 3.0), data2.Rows.Find("Width")["Max"]);
+
+            // Run again to confirm two operations isn't the limit!
+            VerySimpleEstimate.Run(sim).Wait();
+            DataTable data3 = sim.Data;
+            Assert.Equal(2.0 + 3.0 + 3.0, data3.Rows.Find("Width")["Sum"]);
+            Assert.Equal(3.0, data3.Rows.Find("Width")["Max"]);
```

Comment:
These labels change after merging #267

Reply:
n/a anymore as abandoned column name changes in #267